src/Containers/MiddleCanvas/DroppableElement.tsx

```diff
@@ -0,0 +1,107 @@
+    droppableLabels: string[][],
+    isPreview?: boolean,
+};
+
+const FIRST_LABEL = 0;
+
+export const DroppableElement: React.FC<DroppableElementProps> = ({
+    droppableLabels,
+    isPreview,
+    ...props
+}): React.ReactElement => {
+    const [items, setItems] = useState(droppableLabels);
+    
+    /**
+     * Handles the draggable elements when dragged - required function
+     * @param {DropResult} result - react-beautiful-dnd object that gives access to source and destination ids
+     */
+    const onDrag = (result: DropResult): void => {
+        const { source, destination } = result;
+
+        if(!destination) return;
+        
+        const reorderedList = reorder(items, source.index, destination.index);
+        setItems(reorderedList);
+    };
+
+    const renderDraggableComponent = () => droppableLabels.map((droppableLabel, index) => (
+        <Draggable
+            key={droppableLabel[FIRST_LABEL]}
+            draggableId={droppableLabel[FIRST_LABEL]}
+            index={index}
+            isDragDisabled={isPreview}
+        >
+            {(providedDraggable: DraggableProvided, snapshotDraggable: DraggableStateSnapshot) => (
+                <Wrapper
+                    ref={providedDraggable.innerRef}
+                    {...providedDraggable.draggableProps}
+                    {...providedDraggable.dragHandleProps}
+                    style={providedDraggable.draggableProps.style}
+                >
+                    <DroppableContainer isPreview={isPreview} isDragging={snapshotDraggable.isDragging}> 
+                        <DroppableContainerContents droppableLabel={droppableLabel} />
+                    </DroppableContainer>
+                </Wrapper>
+            )}
+        </Draggable>
+    ));
+
+    return (
+        <Wrapper {...props}>
+            <DragDropContext onDragEnd={onDrag}>
+                <Droppable droppableId="templateDroppable">
+                    {(provided: DroppableProvided) => (
+                        <div
+                            ref={provided.innerRef}
+                            {...provided.droppableProps}
+                        >
+                            {renderDraggableComponent()}
+                            {provided.placeholder}
+                        </div>
+                    )}
+                </Droppable>
+            </DragDropContext>
+        </Wrapper>
+    );
+}
+
+const Wrapper = styled.div``;
+
+interface DroppableContainerProps {
+    isDragging?: boolean,
+    isColumn?: boolean,
+    isPreview?: boolean
+};
+const DroppableContainer = styled.div<DroppableContainerProps>`
+    border-radius: 4px;
+    margin: 15px;
+    ${media(
+        'phone',
+        `
+        height: 60px;
+    `)};
+    ${({ theme, isDragging, isPreview }): string => `
+        background-color: ${isPreview ? theme.colors.background : theme.colors.input.default};
```

Comment:
You should group the set of isPreview CSS together instead of one line per each one btw